cli/pkg/kube/client.go

```diff
@@ -45,7 +44,25 @@ type Object interface {
 	runtime.Object
 }
 
-func CreateClient(cmd *cobra.Command) (*Client, error) {
+var cliClient *Client
```

Comment:
I know having globals is convenient, but can be considered problematic for several reasons.